python/ql/src/semmle/python/frameworks/Psycopg.qll

```diff
@@ -0,0 +1,39 @@
+/**
+ * Provides classes modeling security-relevant aspects of the `Psycopg` PyPI package.
+ * See
+ * - https://www.psycopg.org/docs/
+ * - https://pypi.org/project/psycopg2/
+ */
+
+private import python
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.dataflow.new.RemoteFlowSources
+private import semmle.python.Concepts
+private import PEP249
+
+/**
+ * Provides models for the `Psycopg` PyPI package.
+ * See
+ * - https://www.psycopg.org/docs/
+ * - https://pypi.org/project/psycopg2/
+ */
+module Psycopg {
+  // ---------------------------------------------------------------------------
+  // Psycopg
+  // ---------------------------------------------------------------------------
+  /** Gets a reference to the `Psycopg` module. */
+  private DataFlow::Node modulePsycopg(DataFlow::TypeTracker t) {
```

Comment:
In most of our modeling we haven't had this `module` prefix, so let's stick with that for now. The idea being that Python interface maps very closely to the CodeQL interface... so I think we should actually be using the module name here (and I guess that becomes impossible if it starts with an uppercase letter, so not a perfect solution by any means).

```suggestion
  private DataFlow::Node psycopg2(DataFlow::TypeTracker t) {
```